reference/orders.v2.oas2.yml

```diff
@@ -4271,7 +4271,10 @@ components:
           description: A read-only value representing the last modification of the order. Do not attempt to modify or set this value in a POST or PUT request. RFC-2822
         date_shipped:
           type: string
-          description: A read-only value representing the date of shipment. Do not attempt to modify or set this value in a POST or PUT request. RFC-2822
+          description: >
+            A read-only value representing the date the order is fully shipped.
```

Comment:
A read-only value representing the date `when` the order is fully shipped.
I think that reads slightly nicer.
Quick question, is `fully shipped` terminology we use, or do we need to be more descriptive. like `the date when all items in an order have been shipped.`.